tests/llms/test_openai_like.py

```diff
@@ -0,0 +1,98 @@
+from unittest.mock import patch
+
+import pytest
+from llama_index.llms import OpenAILike
+from llama_index.llms.base import ChatMessage
+
+
+def test_interfaces() -> None:
+    llm = OpenAILike(model="placeholder")
+    assert llm.class_name() == type(llm).__name__
+    assert llm.model == "placeholder"
+
+
+def test_completion() -> None:
+    llm = OpenAILike(
+        model="models/placeholder.gguf", is_chat_model=True, context_window=1024
+    )
+
+    text = "...\n\nIt was just another day at the office. The sun had ris"
+    with patch(
+        "llama_index.llms.openai.completion_with_retry",
+        return_value={
+            "id": "123",
+            "object": "text_completion",
+            "created": 1696036786,
+            "model": "models/placeholder.gguf",
+            "choices": [
+                {
+                    "text": text,
+                    "index": 0,
+                    "logprobs": None,
+                    "finish_reason": "length",
+                }
+            ],
+            "usage": {"prompt_tokens": 13, "completion_tokens": 16, "total_tokens": 29},
+        },
+    ) as mock_completion:
+        response = llm.complete(
+            "A long time ago in a galaxy far, far away", use_chat_completions=False
+        )
+    assert response.text == text
+    mock_completion.assert_called_once()
+    # Check we remove the max_tokens if unspecified
+    assert "max_tokens" not in mock_completion.call_args.kwargs
+
+
+def test_chat() -> None:
+    llm = OpenAILike(
+        model="models/placeholder.gguf", globally_use_chat_completions=True
```

Comment:
Are you running `make test`?  I think this test will fail because `globally_use_chat_completions` is not an argument of `OpenAILike`